File: C/Beecrowd/1019.c
#include <stdio.h>
#include <math.h>
int converte(int segundos);


int main(){
    int segundos;
    scanf("%d", &segundos);

    converte(segundos);


    return 0;
}

int converte(int segundos){
    int minutos, horas;

    horas = segundos/(60*60);
    segundos-= (60*60)*horas;
    // se for 0 ele inicializa com 0
    minutos = (segundos/60);
    segundos-= 60 * minutos;


    printf("%d:%d:%d\n", horas, minutos, segundos);
    return 0;
}

File: Python/Beecrowd/1019.py
N = int(input())
horas = 0
minutos = 0
segundos = 0

if((N//3600)!=0):
    horas = N//3600
    N -= horas * 3600
if(N//60!=0):
    minutos = N//60
    N -= minutos * 60

segundos = N

print(f"{horas}:{minutos}:{segundos}")
